### PR TITLE
icons made

### PR DESCRIPTION
**Pull request by harae37**
whoever's working with manifest.js please change

 icon.svg

into

icon-48.png
icon-96.png

### CODE REVIEW

**Comment by nbriz** [comment]
write now no one is working on `manifest.json` (in theory @zylemma && @jwham92  are, but i think they've made all the updates they need to make to that file, at least for now). so i suggest u @harae37 make the change to manifest && push it so it gets included in this pull request, && then i'll accept it

File: addon/manifest.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
   "description": "Track your web usage and generate a beautiful diagram. ",
 
   "icons": {
-    "48": "icons/icon.svg",
-    "96": "icons/icon.svg"
+    "48": "icons/icon-48.png",
+    "96": "icons/icon-96.png"
   },
 
   "browser_action": {
```
Comment (nbriz) [inline addon/manifest.json:14]: almost, except that in ur `browser_action` section ur `default_icon` is still set to the old svg icon
```json
    "default_icon": "icons/icon.svg",
```
it should be set to ur new icon-48.png

**Review by nbriz — CHANGES_REQUESTED** [review]

File: addon/manifest.json (hunk 1 above)
Comment (harae37) [reply addon/manifest.json:14]: Just changed it. Is it automatically going to update the pull request?

**Comment by nbriz** [comment]
yep, it automatically updates the PR.
i just checked the addon, looks good.
merging the pull request now

**Pull request merged**